lib/apiservers/engine/backends/container.go

```diff
@@ -1096,13 +1096,14 @@ func (c *ContainerBackend) containerStart(op trace.Operation, name string, hostC
 		handle = bindRes.Payload.Handle
 		endpoints = bindRes.Payload.Endpoints
 
+		// due to issue 8405, 7128 and 8052, we will not clear out assigned IP address besides container is removed
```

Comment:
This unbind() happens during error, so we should keep the undo here?

Reply:
I assume here we have assigned an IP address here, so we do not release the IP address even though it is not started.

Reply:
Since each exited container would keep an IP, should we inform users to reap exited containers to avoid ip exhausting?

Reply:
We inform users when there is no available ip, see line 164